core/app/components/pubs/PubEditor/actions.db.test.ts

```diff
@@ -110,11 +110,17 @@ describe("updatePub", () => {
 			name: "Can update if all permissions are satisfied",
 			loginUser: { id: crypto.randomUUID() as UsersId },
 			userRole: MemberRole.admin,
-			expected: [
-				{
-					value: "new title",
-				},
-			],
+			expected: {
+				values: [
+					{
+						fieldName: "Title",
+						rank: null,
+						relatedPubId: null,
+						schemaName: "String",
+						value: "new title",
+					},
+				],
+			},
```

Comment:
This test had to be updated because an unfortunate side effect of using pub-op is that we now always call `getPubsWithRelatedValues` and return that after the update. We don't really need that because the pub update page updates from our autoRevalidated queries and we don't even look at the return value of the server action! So it would be nice to make the final query optional and/or figure out a way to use `returning` instead in pub-op eventually!

Reply:
hmm good point! would not be that hard to add a `returning` thing at the end, if it's not there then we don't do the fetch. would you like to see anything in particular returned you think if `returning` is not specified?

Reply:
i guess we could also return the things that are changed, but that might get quite messy